src/App.jsx

```diff
@@ -26,20 +27,15 @@ export const alex = {
 export const App = () => (
   <div className="App">
     <section className="Person">
```

Comment:
`<section className="Person">` should be inside the `Person` component (since it is a Person related info it makes sense to assign the responsibility of rendering it to the Person component)